src/Microsoft.Azure.Relay/HybridConnectionListener.cs

```diff
@@ -994,6 +996,7 @@ public void AcceptSucceeded(HybridConnectionStream connection)
                     clientWebSocket.Options.SetBuffer(this.bufferSize, this.bufferSize);
                     clientWebSocket.Options.SetRequestHeader("Host", this.Address.Host);
                     clientWebSocket.Options.Proxy = this.listener.Proxy;
+                    clientWebSocket.Options.AddSubProtocol(this.subProtocol);
```

Comment:
This would now add a subProtocol of "" (string.Empty) when the original sender didn't specify any subprotocol.  Probably better to have `if (!string.IsNullOrEmpty(this.subProtocol)` surrounding this line.